### PR TITLE
Fix test errors in source tables

### PR DESCRIPTION
**Pull request by chowbao**
Casting closed_at to datetime and fixing mismatching time data types

### CODE REVIEW

File: models/sources/src_history_operations.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ sources:
         tests:
           - dbt_utils.recency:
               datepart: hour
-              field: closed_at
+              field: cast(closed_at as datetime)
```
Comment (sydneynotthecity) [inline models/sources/src_history_operations.yml:12]: does it matter that the partition column for this table is actually `batch_run_date`?

File: models/sources/src_history_transactions.yml [hunk 2]
```diff
@@ -22,25 +22,25 @@ sources:
             tests:
               - unique:
                   config:
-                    where: closed_at >= datetime_trunc(datetime_sub(current_datetime(), interval 2 day), day)
-                            and closed_at < datetime_trunc(current_datetime(), day)
+                    where: cast(closed_at as datetime) >= datetime_trunc(datetime_sub(current_datetime(), interval 2 day), day)
+                            and cast(closed_at as datetime) < datetime_trunc(current_datetime(), day)
```
Comment (sydneynotthecity) [inline models/sources/src_history_transactions.yml:26]: might need to add the partition fields to this where clause otherwise it's gonna be expensive

**Review by sydneynotthecity — APPROVED** [review]
Had a question but doesn't block the merge. I think we'll need to query processing closely after deploy to make sure we're not scanning too much data

File: models/sources/src_history_operations.yml (hunk 1 above)
Comment (chowbao) [reply models/sources/src_history_operations.yml:12]: Hmm you're right it doesn't matter

File: models/sources/src_history_transactions.yml (hunk 2 above)
Comment (chowbao) [reply models/sources/src_history_transactions.yml:26]: Yeah I'll look at these tables again. It should be based on the partitioned field

**Pull request merged**